spec/features/severe-weather/severe-weather.spec.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-env mocha */
+const chai = require('chai');
+const constants = require('./constants');
+const SevereWeatherAutomater = require('./severe-weather.automater');
+
+const expect = chai.expect;
+
+let automater;
```

Comment:
Having worked with functional languages before, it's weird for me to see a "let" keyword without assignment.

I'm not saying it's wrong, it's just weird to me.